computer-vision/README.md

```diff
@@ -0,0 +1,26 @@
+# ArUco Robot Tracking
```

Comment:
So technically not related to this PR

Reply:
Oh oops, forgot to change the content of the README.